services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationArea.kt

```diff
@@ -10,5 +12,19 @@ sealed interface ApplicationArea {
 data class CableReportApplicationArea(override val name: String, override val geometry: Polygon) :
     ApplicationArea
 
-data class ExcavationNotificationArea(override val name: String, override val geometry: Polygon) :
-    ApplicationArea
+data class ExcavationNotificationArea(
+    override val name: String,
+    override val geometry: Polygon,
+    val hankealueId: Int,
+    val tyoalueet: List<Tyoalue>,
+    val katuosoite: String,
+    val tyonTarkoitukset: Set<TyomaaTyyppi>,
+    val lisatiedot: String,
+    val haitat: Haitat,
+) : ApplicationArea
+
+data class Tyoalue(
+    val geometry: Polygon,
+    val area: Double,
+    val haittaindeksi: Float,
```

Comment:
I think this should be a `TormaystarkasteluTulos`.

Reply:
Yes, you are right. There should be more data than just the index. Fixed.